frontend/src/metabase/notifications/AddEditSidebar/AddEditEmailSidebar.tsx

```diff
@@ -80,6 +83,15 @@ export const AddEditEmailSidebar = ({
 }: AddEditEmailSidebarProps) => {
   const isValid = dashboardPulseIsValid(pulse, formInput.channels);
   const userCanAccessSettings = useSelector(canAccessSettings);
+  const currentUser = useSelector(getCurrentUser);
+
+  // This function changes every render because it's produced by _.partial() and isn't memoized.
+  const onChangePropertyChangeRef = useRef(onChannelPropertyChange);
+  useEffect(() => {
+    if (isEmbeddingSdk()) {
+      onChangePropertyChangeRef.current("recipients", [currentUser]);
```

Comment:
I checked that the recipients on the email input are from `/api/user/recipients`, so they have the same shape as the `currentUser` (from user table), so we can just use it here as is.